lib/ui/src/modules/ui/libs/is_mobile_device.js

```diff
@@ -0,0 +1,7 @@
+export default userAgent => {
+  if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(userAgent)) {
```

Comment:
Do we really need javascript for this? Can we just show/hide things using media queries instead?

Reply:
Right now, I can't test it using emulator from devTools

Reply:
I did it to avoid changes to layout in general.
Depending on whether it is mobile device, the header is rendered outside of stories-panel as a standalone component and visibility of stories-panel is just toggled with the existing mechanism. 
Plus for stories-panel as well. To hide it initially for mobile devices (in initial state) :)

Reply:
OK makes sense

Reply:
I'm in the media query camp as well. If on desktop, a user scales the window smaller so they can code and render in the browser side by side, they should be able to use the mobile view as well.

Reply:
@danielduan with current version of Storybook there is nothing stopping users from hiding stories-panel on desktop (with shortcuts) and then resize the browser window as it is needed for your particular case. I want to emphasize that this pull request solves the problem with testing components on physical mobile devices :) 
And I tried to affect as little of desktop version as possible on purpose. Cause this is a whole separate task related to layouts in general.

Reply:
I understand that you don't want to expand the scope of this change so this sounds fine with me.

It's still a broader issue that we'll need to address at some point.

Reply:
Let's take it 1 step at a time? I don't fancy userAgent sniffing either, but it does solve an issue and down-scopes this PR well.

Let's improve the manager head for desktop after.